src/RA_AchievementSet.h

```diff
@@ -27,7 +27,8 @@ class AchievementSet
     void Clear();
     void Test();
 
-    BOOL Serialize(FileStream& Stream);
+    _DEPRECATEDR("Why not submit each ach straight to cloud?")
+    BOOL Serialize(_UNUSED std::iostream& Stream);
```

Comment:
Presently, the compiler won't give you warning. If for some reason someone did use a function marked as deprecated, the compiler will warning that you are using a deprecated function.